demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -181,35 +181,19 @@ export class DemoMeetingApp {
     }
   }
 
-  getAudioInputDevice = async (): Promise<MediaDeviceInfo> => {
-    const audioInputDevices = await this.deviceController.listAudioInputDevices();
+  getAudioInputDevice = async (): Promise<string> => {
```

Comment:
This all gets a lot neater if you make this `Promise<Device>` — or even `Promise<AudioInputDevice>`, if you can swing that.

Reply:
Change to Device